x-pack/plugins/security_solution/public/management/pages/event_filters/view/event_filters_list_page.tsx

```diff
@@ -9,8 +9,24 @@ import React, { memo } from 'react';
 import { i18n } from '@kbn/i18n';
 import { FormattedMessage } from '@kbn/i18n/react';
 import { AdministrationListPage } from '../../../components/administration_list_page';
+import { Empty } from './components/empty';
+import { useEventFiltersNavigateCallback, useEventFiltersSelector } from './hooks';
+import { getCurrentLocation } from '../store/selector';
+import { EventFiltersFlyout } from './components/flyout';
 
 export const EventFiltersListPage = memo(() => {
+  const handleAddButtonClick = useEventFiltersNavigateCallback(() => ({
```

Comment:
I know you probably cloned what was done for Trusted apps wiht this `useEventFiltersNavigateCallback()`, but that hook is really not efficient or memoized because we keep calling the hook with a different callback.  This was also discussed back when that was first introduced, but we did not have time to either address it or refactor it. 

I would suggest us using regular `useCallback()` to memoize these.